### PR TITLE
Bug 1522751, Removed template as the label name

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1522751

### CODE REVIEW

File: dev_guide/application_lifecycle/promoting_applications.adoc [hunk 1]
```diff
@@ -684,7 +684,7 @@ pipeline:
 ----
 $ oc login <source_environment>
 $ oc project <source_project>
-$ oc export dc,is,svc,route,secret,sa -l template=<application_template_name> -o yaml > export.yaml
+$ oc export dc,is,svc,route,secret,sa -l promotion-group=<application_template_name> -o yaml > export.yaml
```
Comment (gabemontero) [inline dev_guide/application_lifecycle/promoting_applications.adoc]: change `application_template_name` to `application_name` as well to be consistent with the instruction above

**Review by gabemontero — CHANGES_REQUESTED** [review]

**Comment by ahardin-rh** [comment]
@gabemontero Thanks! This is updated.

**Review by gabemontero — APPROVED** [review]
/lgtm

**Comment by ahardin-rh** [comment]
QE verified.

**Comment by ahardin-rh** [comment]
No revision history needed

**Pull request merged**